tests/fast-integration/telemetry-test/helpers.js

```diff
@@ -109,3 +110,26 @@ function waitForPodWithLabel(
   );
 }
 
+function waitForDeploymentWithLabel(
```

Comment:
may be we can move this to here: https://github.com/kyma-project/kyma/blob/main/tests/fast-integration/utils/index.js it can be useful for other tests also.

Reply:
moved to the utils